scripts/bootstrap

```diff
@@ -119,6 +126,11 @@ info_msg "Search static folder location"
 static_folder=$(pipenv run invenio shell --no-term-title -c "print('static_folder:%s' % app.static_folder)"|grep static_folder| cut -d: -f2-)
 info_msg "Install static folder npm dependencies in: ${static_folder}"
 pipenv run npm install --prefix "${static_folder}"
+if [ ! -d $TGZ_PATH ]
```

Comment:
What about using `-z` test to check if TGZ_PATH is set? And then `-d`?

Reply:
Because of `tgz_path:` at line 61, a value is expected. If not, there will be an error message saying `missing parameter`. So I think it's redundant. I'll check.

Reply:
Ok, no error message saying `missing parameter`. But if no value is given to TGZ_PATH, the condition is considered as `false` and bootstrap script is run normally.